python/ray/dashboard/subprocesses/utils.py

```diff
@@ -27,12 +31,29 @@ def module_logging_filename(module_name: str, logging_filename: str) -> str:
     EXTENSION = ".log"
     return "dashboard_TestModule.log"
     """
-    if not logging_filename:
-        return "stderr"
     stem, extension = os.path.splitext(logging_filename)
+    if is_stderr:
+        extension = ".err"
     return f"{stem}_{module_name}{extension}"
 
 
+def setup_err_logging(module_name: str, log_dir: str, logging_filename: str):
```

Comment:
We have a function called `configure_log_file` and you can just use that.

Reply:
Updated